src/conn.ts

```diff
@@ -61,7 +53,7 @@ export interface PacketMiddleware {
   (packetData: PacketData): PacketMiddlewareReturnValue | Promise<PacketMiddlewareReturnValue>;
 }
 
-type PacketMiddlewareReturnValue = PacketData | undefined | false;
+type PacketMiddlewareReturnValue = PacketData | undefined | false | true | void;
```

Comment:
what does it do on `void`? maybe add something about this in the readme?

Reply:
When writing functions without a return statement typescript defines the return type as void instead off undefined. There is not void type for js. This is just so typescript does not complain.